k8s/kube-base/mariadb-conf/etc-mysql-conf.d-mysqld_safe_syslog.cnf

```diff
@@ -0,0 +1,4 @@
+# from configmap
```

Comment:
Do you  want to add a similar comment to the other config files?

Reply:
There were similar comments, though they were buried.  It is easier to see if it is the first line.